corehq/apps/data_interfaces/tasks.py

```diff
@@ -89,6 +92,13 @@ def run_case_update_rules_for_domain(domain, now=None):
 
         for case_ids in case_id_chunks:
             for case in CaseAccessors(domain).iter_cases(case_ids):
+                time_elapsed = datetime.utcnow() - start_run
+                if time_elapsed.seconds > ONE_DAY_IN_SECONDS:
+                    notify_error(
+                        "Halting rule run for domain %s as it's been running for more than a day." % domain
```

Comment:
would it be useful to know which rule is being halted so that it can be looked into further? 
Also, won't this rule just get re-run next time the task gets triggered? Or is that the intended behaviour?